src/Kernels/Local.h

```diff
@@ -80,17 +80,17 @@ class seissol::kernels::Local : public LocalBase {
                                         ConditionalIndicesTable& indicesTable,
                                         kernels::LocalData::Loader& loader,
                                         double time,
-                                        double timeStepWidt);
+                                        double timeStepWidth);
 
     void flopsIntegral(FaceType const i_faceTypes[4],
                        unsigned int &o_nonZeroFlops,
                        unsigned int &o_hardwareFlops );
                         
     unsigned bytesIntegral();
 
-    void setDamageParameters( seissol::initializer::parameters::DamagedElasticParameters* iDamagedElasticParameters ) {
-      mDamagedElasticParameters = iDamagedElasticParameters;
-    }
+    void const setDamagedElasticParameters(seissol::initializer::parameters::DamagedElasticParameters* mDamagedElasticParameters){
+      damagedElasticParameters = mDamagedElasticParameters;
```

Comment:
Also here, I suppose to move the `m` prefix, such that the member of the class has the prefix and the parameter does not have the prefix.